src/buildercore/bootstrap.py

```diff
@@ -252,12 +252,17 @@ def _update_ec2_node():
         install_master_flag = str(is_master).lower() # ll: 'true' 
         master_ip = master(region, 'private_ip_address')
 
+        # TODO: this is a little gnarly. I think I'd prefer this logic in the script:
+        #       if [ cat /etc/build-vars.json | grep 'nodename' ]; then ... fi
```

Comment:
it was just pseudo-code ;) 
```
if build-vars exists
    minion_id = 
        open build-vars
        extract and return nodename
otherwise, 
    use given stackname
```

and I was just thinking out loud. We can leave it as-is for now. I put in the `# message` as a message for my future self.